.circleci/config.yml

```diff
@@ -0,0 +1,48 @@
+version: 2
+jobs:
+  build:
+    working_directory: ~/test
+    docker:
+      - image: circleci/node:8.11.1
+    steps:
+      - checkout
+      - run:
+          name: submodule 
+          command: git submodule init && git submodule update 
+      - run:
+          name: update-npm
+          command: 'sudo npm install -g npm@latest'
+      - restore_cache:
+          key: dependency-cache-{{ checksum "package.json" }}
+      - run:
+          name: install
+          command: npm install
+      - save_cache:
+          key: dependency-cache-{{ checksum "package.json" }}
+          paths:
+            - ./node_modules
+      - run:
+          name: build kitsune
+          command: npm run build-kitsune
+      - run:
+          name: test
+          command: npm run test
```

Comment:
This currently doesnt include the test-service npm task, we should add that here as well or add it to the 'test' npm task

Reply:
if the test isnt doing anything interesting, then we can keep only one, either `test` or `service-test`

Reply:
both are useful atm. We should run the functionality of both ```test``` and ```service-test```, either by combining them both into a command - or by just running both commands

Reply:
added both